model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+    metric_name: vcs.repository.count
+    brief: 'The number of repositories in an organization'
+    instrument: gauge
+    unit: "{repository}"
+    stability: experimental
+    attributes: []
+  - id: metric.vcs.repository.ref.count
+    type: metric
+    metric_name: vcs.repository.ref.count
+    brief: 'The number of refs of type branch or tag in a repository'
+    instrument: updowncounter
+    unit: "{ref}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.ref.lines_added
+    type: metric
+    metric_name: vcs.repository.ref.lines_added
+    brief: 'The number of lines added in a ref (branch) relative to the default branch (trunk)'
+    instrument: gauge
+    unit: "{line}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.repository.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.ref.lines_deleted
+    type: metric
+    metric_name: vcs.repository.ref.lines_deleted
+    brief: 'The number of lines deleted in a ref (branch) relative to the default branch (trunk)'
+    instrument: gauge
+    unit: "{line}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.repository.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.ref.revisions_ahead
+    type: metric
+    metric_name: vcs.repository.ref.revisions_ahead
+    brief: 'The number of revisions (commits) a ref (branch) is ahead of the default branch (trunk)'
+    instrument: gauge
+    unit: "{revision}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.repository.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.ref.revisions_behind
+    type: metric
+    metric_name: vcs.repository.ref.revisions_behind
+    brief: 'The number of revisions (commits) a ref (branch) is behind the default branch (trunk)'
+    instrument: gauge
+    unit: "{revision}"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.repository.ref.type
+        requirement_level: required
+  - id: metric.vcs.repository.ref.time
+    type: metric
+    metric_name: vcs.repository.ref.time
+    brief: 'Time a ref (branch) created from the default branch (trunk) has existed. The `ref.type` attribute will always be `branch`'
```

Comment:
Should we instead report creation time for each branch? Then people can derive other things including deltas from it. It could also be useful for tags.  E.g. `vcs.ref.created|creation_time`

Reply:
@adrielp what's your opinion?

Reply:
Currently it's a duration metric.

It's not possible to get the creation date from the Gihtub/Gitlab API.
So the commit time of the earliest 

Code: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/githubreceiver/internal/scraper/githubscraper/helpers.go#L257

Advantage of using creation time is that it mostly stays constant (except for rebase).